src/commands/Tools/Websearch/igdb.ts

```diff
@@ -144,4 +143,24 @@ export default class extends RichDisplayCommand {
 		if (!platforms || isArrayOfNumbers(platforms)) return fallback;
 		return platforms.map((platform) => platform.name).join(', ');
 	}
+
+	private buildBody(game: string) {
+		return [
+			[`search "${game}"`, 'limit 10', 'offset 0;'].join('; '),
+			`fields ${[
+				'name',
+				'url',
+				'summary',
+				'rating',
+				'involved_companies.developer',
+				'involved_companies.company.name',
+				'genres.name',
+				'release_dates.date',
+				'platforms.name',
+				'cover.url',
+				'age_ratings.rating',
+				'age_ratings.category;'
+			].join(',')}`
+		].join(' ');
```

Comment:
```suggestion
				[
					`search: "${game}"`,
					`fields ${[
						'name',
						'url',
						'summary',
						'rating',
						'involved_companies.developer',
						'involved_companies.company.name',
						'genres.name',
						'release_dates.date',
						'platforms.name',
						'cover.url',
						'age_ratings.rating',
						'age_ratings.category'
					].join(',')}`,
					'limit 10',
					'offset 0'
				].join('; ')
```

This way we only need 2 array joins.

Outputs:
```
search: "zelda"; fields name,url,summary,rating,involved_companies.developer,involved_companies.company.name,genres.name,release_dates.date,platforms.name,cover.url,age_ratings.rating,age_ratings.category; limit 10; offset 0
```